drivers/rke/rke_driver.go

```diff
@@ -180,6 +181,10 @@ func getData(s Store, k8sVersion string) map[string]interface{} {
 	if data2 != nil {
 		data["k8s-service-options"] = data2
 	}
+	data3 := s.GetWindowsServiceOptions(k8sVersion)
+	if data3 != nil {
+		data["k8s-windows-service-options"] = data3
```

Comment:
corresponding to this, RKE needs to read this from `k8s-windows-service-options`, which is done here.. https://github.com/rancher/rke/blob/master/cmd/up.go#L89 

and it gets read while building kubelet, kubeApi etc when passed in data. 
Here's an example of it reading from Kubelet: 
https://github.com/rancher/rke/blob/master//cluster/plan.go#L444

Reply:
Thanks, for now, we don't support to use `rke up` to set up a Windows cluster, it only could be created by rancher server. The purpose of placing parameters/options/images into metadata is more convenient for future unified management.

Reply:
It's not actually for `rke up`, but for rancher. When a new kubernetes version gets added with some new service options for windows in `kontainer-driver-metadata` and rancher wants to send that information to rke...we pass these options to rke. 

RKE needs to be able to read them, the line points to `ClusterUp` which is called by rancher as well as cmd. So if data has any value, we need to use that else we'll fall back to default ones.